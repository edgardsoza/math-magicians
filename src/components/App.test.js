import React from 'react';
import renderer from 'react-test-renderer';
import Mainpage from './mainpage';
import Quote from './quotes';
import Calculator from './calculator';

describe("testing the components", () => {
  it('renders mainpage correctly', () => {
    const tree = renderer
      .create(<Mainpage />)
      .toJSON;
    expect(tree).toMatchSnapshot();  
  });

  it('renders mainpage correctly', () => {
    const tree = renderer
      .create(<Quote />)
      .toJSON;
    expect(tree).toMatchSnapshot();  
  });
  
  it('renders calculator correctly', () => {
    const tree = renderer
      .create(<Calculator />)
      .toJSON;
    expect(tree).toMatchSnapshot();  
  });
})
