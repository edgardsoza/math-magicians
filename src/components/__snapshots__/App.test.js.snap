// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the components renders calculator correctly 1`] = `[Function]`;

exports[`testing the components renders mainpage correctly 1`] = `[Function]`;

exports[`testing the components renders mainpage correctly 2`] = `[Function]`;
